gce/dns-records.tf

```diff
@@ -45,3 +45,11 @@ resource "google_dns_record_set" "docker-registry" {
   ttl = "60"
   rrdatas = ["${google_compute_instance.docker-registry.network_interface.0.address}"]
 }
+
+resource "google_dns_record_set" "postgresapi" {
+  managed_zone = "${var.dns_zone_id}"
+  name = "${var.env}-postgresapi.${var.dns_zone_name}"
+  type = "A"
```

Comment:
On GCE though, *.<env>-hipaiche points to the CNAME of ${google_dns_record_set.router.name}... so my question still stands